caravel/assets/javascripts/explorev2/components/SelectField.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes } from 'react';
+import { FormGroup, FormControl } from 'react-bootstrap';
+import ControlLabelWithTooltip from './ControlLabelWithTooltip';
+
+const propTypes = {
+  label: PropTypes.string,
+  description: PropTypes.string,
+};
+
+const defaultProps = {
+  label: null,
+  description: null,
+};
+
+export default function SelectField({ label, description }) {
+  return (
+    <FormGroup controlId={`formControlsSelect-${label}`}>
+      <ControlLabelWithTooltip label={label} description={description} />
+      <FormControl componentClass="select" placeholder="select">
+        <option value="select">select</option>
```

Comment:
do we want this instead of react-select? I think multi option with react-select looks better than the multi-select of FormControl in react-bootstrap